java-example-base-image/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM openjdk:8u252-slim-buster
+RUN apt-get update && apt-get install -y  curl
+CMD java ${JAVA_OPTS:-"-Xmx64m"} -jar ${JAR:-"application.jar"}
```

Comment:
service.jar would be a better name :-)
You could also add HEALTHCHECK?

Reply:
will do